tests/runfiles/linux.run

```diff
@@ -348,6 +348,9 @@ tests = ['zpool_upgrade_001_pos',
     'zpool_upgrade_006_neg', 'zpool_upgrade_008_pos',
     'zpool_upgrade_009_neg']
 
+[tests/functional/cli_root/zpool_sync]
+tests = ['zpool_sync_001_pos', 'zpool_sync_002_neg']
```

Comment:
I've updated those but my testing rig is broken so let's see what build bot thinks